File: 1546.c
// 1546 C

#include <stdio.h>

int    main(void)
{
	float	n;
	float	point[1000];
	float	max = 0;
	float	total = 0;

	scanf_s("%f", &n);
	for (int i = 0; i < n; i++)
		scanf_s("%f", &point[i]);
	for (int i = 0; i < n; i++)
	{
		if (max < point[i])
			max = point[i];
	}
	for (int i = 0; i < n; i++)
		total += point[i] / max * 100;
	printf("%f", total / n);
}

File: 1546.py
# 1546 py

total = 0
max = 0
arr = []
n = int(input())
arr = list(map(int, input().split()))
for i in arr:
    if max < i:
        max = i
for i in arr:
    total += i / max * 100
print(total / n)
